src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -471,6 +480,293 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     report_io();
 }
 
+void ov::npuw::CompiledModel::CompiledModelDesc::serialize(std::ostream& stream,
+                                                           const std::size_t& idx,
+                                                           const std::string& device) const {
+    using namespace ov::npuw::s11n;
+
+    static std::string filename = "/home/alexeysm/npuw/tmp_subs/";
+    static std::size_t fid = 0;
+
+    LOG_DEBUG("Serializing CompiledModelDesc...");
+    LOG_BLOCK();
+
+    NPUW_ASSERT(device == "CPU" || device == "NPU");
+    write(stream, device);
+
+    write(stream, replaced_by);
+    // FIXME: is this check enough? E.g. what about FUNCALL_FOR_ALL?
+    if (replaced_by == idx) {
+        NPUW_ASSERT(compiled_model);
+        // !!! FIXME: for some reason cannot import anything after the first submodel
+        // if it's in the same stream
+        std::ofstream fout(filename + "openvino_model" + std::to_string(fid++) + ".blob",
```

Comment:
Issue described in the comments above, need to find a way to handle a single stream (stringstream/resetting stream pointer, etc)